library/ecdsa.c

```diff
@@ -287,31 +287,46 @@ int mbedtls_ecdsa_verify( mbedtls_ecp_group *grp,
 #endif /* MBEDTLS_ECDSA_VERIFY_ALT */
 
 /*
- * Convert a signature (given by context) to ASN.1
+ * Convert a signature (given by context) to ASN.1.
+ * This function may leave a half-written upon encountering an error, and
```

Comment:
I think `buffer` is missing here? `This function is for internal use only. Upon an error, it may leave the buffer partially written.`.

Reply:
Reworded as suggested.